apps/api/v2/src/modules/users/users.repository.ts

```diff
@@ -177,6 +177,19 @@ export class UsersRepository {
     });
   }
 
+  async findAllManagedUsers(oauthClientId: string) {
```

Comment:
We have users directly available on the oauth client, so I think faster query is for you to
```
const client = await this.dbRead.prisma.platformOAuthClient.findUnique({
      where: { id: oauthClientId },
      include: {
        users: true,
      },
    });
    return client?.users ?? [];
```